site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -257,6 +257,37 @@ paths:
             403:
                $ref: '#/components/responses/NotAllowed'
 
+      get:
+         tags:
+            - devices
+         summary: Retrieve all devices for a tenant
+         operationId: getAllDevicesForTenant
```

Comment:
I think the naming of this is misleading. It is not returning "all devices". I think "search" would make more sense. Same for the "summary".

Reply:
I updated the name to `searchDevicesForTenant`